### PR TITLE
SIT-3048 template-db2sfdc-user-migration - implemented munit test

### PR DESCRIPTION
**Pull request by mariandug**

### CODE REVIEW

File: src/test/munit/test-suite.xml [hunk 1]
```diff
@@ -101,11 +99,10 @@ output application/java
 	</sub-flow>
 	<munit:after-test name="test-suiteAfter_Test"
 		description="After tests actions" doc:id="5b85bb9e-67e6-47f7-8f71-53a39f147fc8">
-		<db:delete doc:name="Delete User from DB" doc:id="6aff5b09-2af5-4627-8aa7-a0ac361bbd81"
-			config-ref="Database_Config">
-			<db:sql>DELETE FROM User WHERE email LIKE '#[vars.UserEmail]'</db:sql>
+ 		<db:delete doc:name="Copy_of_Delete User from DB" doc:id="676204e7-0a14-4900-a751-71aba3b97960" config-ref="Database_Config">
```
Comment (ICfl) [inline src/test/munit/test-suite.xml]: wrong name

File: pom.xml [hunk 2]
```diff
@@ -24,9 +24,32 @@
                             <artifactId>mysql-connector-java</artifactId>
                         </sharedLibrary>
                     </sharedLibraries>
-                    <classifier>mule-application-template</classifier>
+                    <classifier>mule-application</classifier>
```
Comment (ICfl) [inline pom.xml]: classifier should be mule-application-template

File: src/test/munit/test-suite.xml (hunk 1 above)
Comment (mariandug) [reply src/test/munit/test-suite.xml]: fixed

File: pom.xml (hunk 2 above)
Comment (mariandug) [reply pom.xml]: changed

**Pull request merged**